account_financial_report/wizard/open_items_wizard.py

```diff
@@ -44,14 +44,32 @@ class OpenItemsReportWizard(models.TransientModel):
     partner_ids = fields.Many2many(
         comodel_name='res.partner',
         string='Filter partners',
+        default=lambda self: self._default_partners(),
     )
     foreign_currency = fields.Boolean(
         string='Show foreign currency',
         help='Display foreign currency for move lines, unless '
              'account currency is not setup through chart of accounts '
-             'will display initial and final balance in that currency.'
+             'will display initial and final balance in that currency.',
+        default=lambda self: self._default_foreign_currency(),
     )
 
+    def _default_foreign_currency(self):
+        return self.env.user.has_group('base.group_multi_currency')
+
+    def _default_partners(self):
+        context = self.env.context
+
+        if context.get('active_ids') and context.get('active_model') \
```

Comment:
The same about how to get partners through recordsets

Reply:
Done